sentences/sv/light_HassLightSet.yaml

```diff
@@ -3,18 +3,18 @@ intents:
   HassLightSet:
     data:
       - sentences:
-          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[s | n | ns ] [<ljusintensitet>] till <brightness>"
-          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[n]  till <brightness> [<ljusintensitet>]"
+          - "[sätt | vrid | dra [på] | skruva | dimm[r]a | ändra ] [upp | ner ] <name>[s | n | ns ] [<ljusintensitet>] till <brightness>"
+          - "[sätt | vrid | dra [på] | skruva | dimm[r]a | ändra ] [upp | ner ] <name>[n]  till <brightness> [<ljusintensitet>]"
           - "<name>[s | n | ns ] [upp | ner ] [till] <brightness> [<ljusintensitet>]"
       - sentences:
-          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <alla> <ljuskällor> [<i_på>] <area>[et] [upp | ner ] till <brightness> [<ljusintensitet>]"
+          - "[sätt | vrid | dra [på] | skruva | dimm[r]a | ändra ] [upp | ner ] <alla> <ljuskällor> [<i_på>] <area>[et] [upp | ner ] till <brightness> [<ljusintensitet>]"
         slots:
           name: "all"
 
       - sentences:
           - "(färga | ändra | byt) <name> [till] {color}"
 
       - sentences:
-          - "{color} ljus i <area>"
+          - "[sätt | ändra ] {color} ljus i <area>"
```

Comment:
I think that we might want some test for this.
I would probably used the sentence:
`ändra till rött ljus i hallen` so maybe add [till] in sentences.

Reply:
Yeah, I saw that after I had saved already and just wanted something to get added. Didn't want to mess with it again, but I can add it. Hopefully.